config/helpers.php

```diff
@@ -413,46 +414,38 @@ function option(string $key, $default = null)
 
 if (Helpers::hasOverride('page') === false) { // @codeCoverageIgnore
     /**
-     * Fetches a single page or multiple pages by
-     * id or the current page when no id is specified
+     * Fetches a single page by id or
+     * the current page when no id is specified
      *
-     * @param string|array ...$id
-     * @return \Kirby\Cms\Page|\Kirby\Cms\Pages|null
-     * @todo reduce to one parameter in 3.7.0 (also change return and return type)
+     * @param string|null $id
+     * @return \Kirby\Cms\Page|null
      */
-    function page(...$id)
+    function page($id = null)
     {
         if (empty($id) === true) {
             return App::instance()->site()->page();
         }
 
-        if (count($id) > 1) {
-            // @codeCoverageIgnoreStart
-            Helpers::deprecated('Passing multiple parameters to the `page()` helper has been deprecated. Please use the `pages()` helper instead.');
-            // @codeCoverageIgnoreEnd
-        }
-
-        return App::instance()->site()->find(...$id);
+        return App::instance()->site()->find($id);
     }
 }
 
 if (Helpers::hasOverride('pages') === false) { // @codeCoverageIgnore
     /**
-     * Helper to build page collections
+     * Helper to build pages collection
      *
      * @param string|array ...$id
-     * @return \Kirby\Cms\Page|\Kirby\Cms\Pages|null
-     * @todo return only Pages|null in 3.7.0, wrap in Pages for single passed id
+     * @return \Kirby\Cms\Pages|null
      */
     function pages(...$id)
     {
-        if (count($id) === 1 && is_array($id[0]) === false) {
-            // @codeCoverageIgnoreStart
-            Helpers::deprecated('Passing a single id to the `pages()` helper will return a Kirby\Cms\Pages collection with a single element instead of the single Kirby\Cms\Page object itself - starting in 3.7.0.');
-            // @codeCoverageIgnoreEnd
+        $pages = App::instance()->site()->find(...$id);
```

Comment:
I think the following should have the same result without the manual creation of the `Pages` instance:

```php
// ensure that a list of string arguments and an array
// as the first argument are treated the same
if (count($id) === 1 && is_array($id[0]) === true) {
    $id = $id[0];
}

// always pass an array to ensure we get a
// collection even if only one ID is passed
return App::instance()->site()->find($id);
```

Reply:
think you are missing something, like an `A::wrap()` or something else you had in mind? Cause the comment says `always pass an array` - but `$id` could still be just a string which would return a single page, wouldn't it?

Reply:
No, since the argument is defined as `...$id` in the function definition, `$id` will always be an array, even if just a single string argument is passed.

Reply:
Ahh, I wasn't aware of that. Then great!